selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java

```diff
@@ -125,35 +122,27 @@ public void createProjectTest() {
     notificationsPopupPanel.waitPopupPanelsAreClosed();
   }
 
-  @Test(priority = 1, groups = FLAKY)
+  @Test(priority = 1)
   public void deleteProjectsFromDashboardTest() {
-    switchToWindow(dashboardWindow);
+    seleniumWebDriver.switchTo().window(dashboardWindow);
     dashboard.selectWorkspacesItemOnDashboard();
     workspaces.selectWorkspaceItemName(WORKSPACE);
     workspaceDetails.selectTabInWorkspaceMenu(PROJECTS);
-    workspaceProjects.waitProjectIsPresent(WEB_JAVA_SPRING);
-    workspaceProjects.waitProjectIsPresent(CONSOLE_JAVA_SIMPLE);
-    openProjectSettings(WEB_JAVA_SPRING);
-    workspaceProjects.clickOnDeleteProject();
-    workspaceProjects.clickOnDeleteItInDialogWindow();
-    workspaceProjects.waitProjectIsNotPresent(WEB_JAVA_SPRING);
+
+    deleteProject(WEB_JAVA_SPRING);
```

Comment:
shouldn't we wait on `WEB_JAVA_SPRING` project is not present after deletion as well? `workspaceProjects.waitProjectIsNotPresent(WEB_JAVA_SPRING);`

Reply:
Project deleting is checked at the end of ```deleteProject()``` method.

Reply:
I see.